source/handshake.c

```diff
@@ -22,20 +22,28 @@
 #include "perflib/perflib.h"
 
 #define NUM_CALLS_PER_TEST        10000
+#define OSSL_LIB_CTX_POOL_SIZE    16
```

Comment:
Should the size of the pool be configurable?

Reply:
It's configurable now.